src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -141,8 +142,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('create', 'acr_build_task_create')
         g.command('show', 'acr_build_task_show')
         g.command('list', 'acr_build_task_list', table_transformer=build_task_output_format)
+        g.command('list-arguments', 'acr_build_arguments_list', client_factory=cf_acr_build_steps)
+        g.command('list-source-properties', 'acr_build_source_repository_properties_list')
         g.command('delete', 'acr_build_task_delete')
         g.command('update', 'acr_build_task_update')
+        g.command('update-build', 'acr_build_task_update_build', client_factory=cf_acr_builds)
```

Comment:
nit: group this with `list-builds`, `show-build`, etc.

Reply:
Changed.